### DISCUSSION TITLE
CreateTaskUC: Assigning study Techniques

### DISCUSSION THREAD

**Discussion by serenahonekin** [CSC207-2022F-UofT/course-project-my-dog-ate-my-hw#32] [General]
For CreateTask use case, I am supposed to assign a task to a default study technique according to their assignment type. 

Should assignment types and their respective study techniques be an enum entity then?

**Comment by XinyingXu** [comment]
I was thinking about static String attribute, but whatever works for you.

**Reply by stephaniesaab** [reply]
A static string may cause issues as it allows for it to be changed later, which we want to avoid since the name of the Task object should change, not the name of the Task class.

**Comment by stephaniesaab** [comment]
I think they should be enum entities since they will have a limited number of characters anyways. The names won't need to be changed anyways as we have the 'custom task' option for that purpose.
